code/modules/research/nanites/program_disks.dm

```diff
@@ -12,6 +12,12 @@
 	if(program_type)
 		program = new program_type
 
+/obj/item/disk/nanite_program/mind_control
+	name = "suspicious-looking disk"
+	desc = "A sinisterly colored nanite program disk. Comes preloaded with a brainwashing program."
+	icon_state = "datadisk_syndie"
+	program_type = /datum/nanite_program/comm/mind_control
```

Comment:
it's really gonna suck to accidentally write over this program

Reply:
I put a warning in the uplink screen